service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersController.java

```diff
@@ -87,8 +93,23 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody Call
 
             directions.forEach((key, value) -> caseDetails.getData().put(key.getValue(), value));
 
-            caseDetails.getData()
-                .put("judgeAndLegalAdvisor", caseData.getStandardDirectionOrder().getJudgeAndLegalAdvisor());
+            caseDetails.getData().put(JUDGE_AND_LEGAL_ADIVSOR_KEY,
+                caseData.getStandardDirectionOrder().getJudgeAndLegalAdvisor());
+        }
+
+        if (isNotEmpty(caseData.getAllocatedJudge())) {
+            JudgeAndLegalAdvisor judgeAndLegalAdvisor;
```

Comment:
could we do JudgeAndLegalAdvisor judgeAndLegalAdvisor = JudgeAndLegalAdvisor.builder().build(); here and remove the need for the else block?